edscrapers/tools/dashboard/pages/insights.py

```diff
@@ -171,27 +173,30 @@ def resources_by_domain_table(self):
         """ function is used to create DataTable containing
         the number of resources/domains """
 
-        # to perform this tasks, we need to collect stas from 2 different sheets
-        # and unifiy it into one dataframe
-
-        # get the resources collected from the datopian end of the air-datopian intersect
-        #df = self._get_df_from_excel_sheet('RESOURCE COUNT PER DOMAIN (DATOPIAN-AIR INTERSECTION)')
-
-        # create a dataframe to hold the necessary info we use for this task
-        #working_df1 = pd.DataFrame(columns=['domain'])
-        #working_df1['domain'] = df['domain']
-        #working_df1['resource count'] = df['resource count_datopian']
-
-        # get the resources from the DAtopian only resource count
-        #df = self._get_df_from_excel_sheet('RESOURCE COUNT PER DOMAIN (DATOPIAN ONLY)')
-
-        #working_df2 = pd.DataFrame(columns=['domain'])
-        #working_df2['domain'] = df['domain']
-        #working_df2['resource count'] = df['resource count']
-
-        # concatenate the 2 dataframes
-        #working_df1 = pd.concat([working_df1,
-        #                        working_df2], axis='index', ignore_index=True)
+        # df = self.resources_by_domain_df()
```

Comment:
This is the refactored code but I had to comment it to comply with the changes Higor made.